parent/pom.xml

```diff
@@ -337,6 +337,24 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.jboss.tools.tycho-plugins</groupId>
+				<artifactId>repository-utils</artifactId>
+				<version>${jbossTychoPluginsVersion}</version>
+				<executions>
+					<execution>
+						<goals>
+							<goal>compare-version-with-baselines</goal>
+						</goals>
+						<phase>verify</phase>
+						<configuration>
+							<baselines>
+								<repo>http://download.jboss.org/jbosstools/updates/stable/luna</repo>
```

Comment:
this should be a property to be possible to test against other baselines.

otherwise this idea looks good -especially if the overhead for checking it in the full build is not big (Timewise)